staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -46,7 +50,36 @@ var x509InsecureSHA1Counter = metrics.NewCounter(
 	},
 )
 
+var (
+	extApiserverAttemptsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "extension_apiserver_attempts_total",
+			Help:           "Counter of extension apiserver attempts broken down by result. It can be either 'OK', 'Not Found', 'Service Unavailable' or 'Internal Server Error'.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
+	)
+
+	extApiserverLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Name:           "extension_apiserver_duration_seconds",
```

Comment:
`request_duration_seconds`

Reply:
Haha, in fact, I also have this habit, but it was written by others with reference to webhook before, and now I have changed it, thanks.